transformer_engine/common/CMakeLists.txt

```diff
@@ -30,7 +30,9 @@ add_library(transformer_engine SHARED
                                layer_norm/ln_api.cpp
                                layer_norm/ln_bwd_semi_cuda_kernel.cu
                                layer_norm/ln_fwd_cuda_kernel.cu
-                               util/cast.cu)
+                               util/cast.cu
+                               fused_softmax/scaled_masked_softmax_cuda.cu
+                               fused_softmax/scaled_upper_triang_masked_softmax_cuda.cu)
```

Comment:
I would probably drop the `_cuda` from the name, considering it is `.cu` file already :-)